src/views/ServiceRequest.js

```diff
@@ -32,15 +32,63 @@ const styles = {
 class ServiceRequest extends Component {
   constructor (props) {
     super(props)
+    this.singleLineFields = ['Name', 'Email', 'Phone', 'Department']
+    this.multiLineFields = [
+      'Project Description',
+      'Project Goal',
+      'Project Budget',
+      'Key Messages',
+      'Primary Target Audience',
+      'Secondary Target Audience',
+      'Project Contact (if other than yourself)',
+      'Comments'
+    ]
+    this.leftCheckboxes = [
+      'Simple1',
+      'Simple2',
+      'Simple3',
+      'Simple4',
+      'Simple5',
+      'Simple6'
+    ]
+    this.rightCheckboxes = [
+      'Simple7',
+      'Simple8',
+      'Simple9',
+      'Simple10',
+      'Simple11',
+      'Simple12'
+    ]
+    const stringProps = [
+      ...this.singleLineFields,
+      ...this.multiLineFields
+    ].reduce(
+      (acc, label) => ({
+        [this.formatLabelToProperty(label)]: '',
+        ...acc
+      }),
+      {}
+    )
```

Comment:
You seemed like you didn't really understand these reduce functions today, so, I thought I would take a stab at explaining. 

A basic `Array#reduce` takes a function and a starting value, in this case, the empty object `{}`. The function gets passed an accumulator (acc in the above code) and the current element of the Array (label in this case) as the first two arguments. This is just using computed properties to format them from the list of fields and set them to an empty string and using object spread (`...acc`) to add the values that have been accumulated. This is all operating over the temporary array that is spreading the two arrays into itself. Once it has looped over all the values in singleLineFields and multiLineFields we are left with stringProps which is an object that contains all the properties we want to put in the state with an empty string initial value.

So a simpler use of reduce might look like this:
```js
[2,2,2,2,2].reduce( (acc, val) => acc + val, 0 ) // returns 10
```